hasjob/views/index.py

```diff
@@ -179,12 +179,8 @@ def fetch_jobposts(request_args, request_values, is_index, board, board_jobs, gk
         data_filters['archive'] = True
         statuses = POSTSTATUS.ARCHIVED
 
-    search_domains = None
-    if search_query:
-        data_filters['query'] = search_query
-        search_domains = Domain.query.filter(
-            Domain.search_vector.match(search_query, postgresql_regconfig='english'), Domain.is_banned == False).options(
-            db.load_only('name', 'title', 'logo_url')).all()  # NOQA
+    if query_string:
+        data_filters['query'] = query_string
```

Comment:
Insert a comment somewhere:
* `query_string` is what the user typed.
* `search_query` is processed for compatibility with PostgreSQL syntax.

Reply:
Added a comment.